cheroot/workers/threadpool.py

```diff
@@ -147,8 +146,13 @@ class ThreadPool:
     """
 
     def __init__(
-            self, server, min=10, max=-1, accepted_queue_size=-1,
-            accepted_queue_timeout=10,
+        self,
```

Comment:
The indentation is broken. It shouldn't match the method body nesting.

Reply:
Again seems like flake8 autoformatter. Do we have any specific autoformatter that we recommend or use?

Reply:
Done.